src/UseScrollIntoView.re

```diff
@@ -0,0 +1,28 @@
+open Belt;
+
+let use = dependency => {
+  let elemRef = React.useRef(Js.Nullable.null);
+  let firstUpdate = React.useRef(true);
+
+  React.useEffect2(
+    () => {
+      let scrollIntoView = () => {
+        elemRef.current
+        ->Js.toOption
+        ->Option.forEach(elem =>
+            elem->Bindings.Element.scrollIntoView({
+              "behavior": "auto",
+              "block": "nearest",
+            })
+          );
+      };
+      // Need to skip the first scroll, to prevent scrolling on receiving initial dependency
```

Comment:
I don't get this part. 

- What is `dependency` for? A code comment to explain would be helpful
- We have to do this dance because of `dependency`, but then, in the usage of the hook, `dependency` is an empty array. Why not have this version stay without `dependency`, and sidestep all the stuff with `firstUpdate`, then have another version `useWithDependency` that gets all the complexity?